.github/policies/resourceManagement.yml

```
@@ -0,0 +1,83 @@
+id: 
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
+owner: 
+resource: repository
+disabled: false
+where: 
+configuration:
+  resourceManagementConfiguration:
+    scheduledSearches: []
+    eventResponderTasks:
+    - if:
+      - payloadType: Pull_Request
+      then:
+      - if:
+        - includesModifiedFiles:
+            files:
+            - SPECS/
+            - SPECS-SIGNED/

```

Should we maybe add `SPECS-EXTENDED` ?
```suggestion
            - SPECS-SIGNED/
            - SPECS-EXTENDED/

```

---

Agreed, but I would make any changes in a separate PR. Let's make sure the gitops thing starts off functional before we introduce our own changes.